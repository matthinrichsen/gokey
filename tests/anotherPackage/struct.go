package anotherPackage

type LastStruct struct {
	Int int
}
