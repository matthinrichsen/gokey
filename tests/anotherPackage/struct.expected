package anotherPackage

type LastStruct struct {
	Int int
}

var abc = LastStruct{Int: 1}
